robotpy_ext/common_drivers/navx/ahrs.py

```diff
@@ -275,6 +275,8 @@ def zeroYaw(self):
         subtracted from subsequent yaw values reported by
         the getYaw() method.
         """
+        if 'disabled' not in inspect.stack()[1][3]:
```

Comment:
This is not a good idea. Instead, ask the DS whether the robot is in disabled mode?

Reply:
ooh I like that